aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/StartupWithRazorPagesAtContentRoot.cs

```diff
@@ -20,8 +20,8 @@ public StartupWithRazorPagesRoot(IConfiguration configuration)
         #region snippet
         public void ConfigureServices(IServiceCollection services)
         {            
-            services.AddRazorPages()
-                .AddRazorPagesOptions(options =>
+            services
+                .AddRazorPages(options =>
```

Comment:
I've bumped this down a line so it can line up with the `.WithRazorPagesAtContentRoot` that follows. I'm not sure I like this formatting, though, so let me know if you usually do it differently for something like this. I couldn't find an exsiting example.

Reply:
We usually have the first one inline (on `services`) and then indent thereafter.